mimic/rest/identity_api.py

```diff
@@ -382,6 +385,7 @@ def validate_token(self, request, token_id):
                 impersonator_session.username)
 
         if token_id in get_presets["identity"]["token_fail_to_auth"]:
+            # weird mix between an unauthorized (401) and not_found (404)
```

Comment:
The response structure being returned is that of a `404`, but the status code is that of a `401`. Why? I don't know; an old OpenStack bug? insight would be appreciated; I'm just highlighting what's is going on. I'll update the comment and if any insights provided I'll add them.